### ISSUE TITLE
Support running from WSL

### ISSUE THREAD

**Issue by mbomb007** [m-ender/vscode-retinate#10]
VS Code is moving towards favoring Windows Subsystem for Linux (WSL) for certain development environments.

If I set `"retinate.retinaPath": "/mnt/c/Program Files/Retina/Retina.exe"`, I get the following error:

```
Command failed: /mnt/c/Program Files/Retina/Retina.exe /tmp/tmp-116s5iZ50fwHyAc.ret

Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\tmp\tmp-116s5iZ50fwHyAc.ret'.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks)
   at System.IO.File.InternalReadAllText(String path, Encoding encoding)
   at System.IO.File.ReadAllText(String path)
   at Retina.Retina.ReadSources(String[] args) in /home/travis/build/m-ender/retina/Retina/Retina/Retina.cs:line 43
   at Retina.Retina.Main(String[] args) in /home/travis/build/m-ender/retina/Retina/Retina/Retina.cs:line 19
```

Could it be that the extension has a hardcoded temp location, rather than using the system's configured temp path?

https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettemppath?view=netcore-2.0

**Comment by m-ender** [comment]
Hm, I'm not sure I'll be able to dig into this any time soon, but it would certainly be good to fix this. Would be happy to review and accept a PR though.

**Comment by mbomb007** [comment]
Actually, the PR would have to go in https://github.com/m-ender/retina, I think, because the arguments were passed correctly to Retina.exe.

I don't know enough C# to know what a solution would be.

However, the main problem is that I was trying to run the Windows version from WSL (Linux). Maybe this can be fixed to work on both, but I got it working by installing dotnet and Retina's Linux version in WSL.

Links I used:
- https://versionsof.net/core/2.2/2.2.101/
  - Must be installed in /usr/share/dotnet, or use the DOTNET_ROOT environment variable
- https://github.com/m-ender/retina/releases/tag/v1.2.0
